pwa/src/templates/cronjobsDetailTemplate/CronjobsDetailsTemplate.tsx

```diff
@@ -22,10 +23,31 @@ export const CronjobsDetailTemplate: React.FC<CronjobDetailPageProps> = ({ cronj
 
   return (
     <Container layoutClassName={styles.container}>
-      {getCronjob.isLoading && <Skeleton height="200px" />}
       {getCronjob.isError && "Error..."}
 
       {getCronjob.isSuccess && <EditCronjobFormTemplate cronjob={getCronjob.data} {...{ cronjobId }} />}
+      {getCronjob.isLoading && <Skeleton height="200px" />}
+
+      <div className={styles.tabContainer}>
```

Comment:
We won't need this container here. The table gets its margin from the main container. The reason we used this container for the tabs is that the tabs didn't inherit the margin from the main container.

And if we use a container we should change its name since this is a table not some tabs. so it would have been `styles.tableContainer`